onnxruntime/core/framework/session_options.h

```diff
@@ -112,7 +112,7 @@ struct SessionOptions {
   std::unordered_map<std::string, const OrtValue*> initializers_to_share_map;
 
   // See onnxruntime_c_api.h for detailed documentation.
-  Status AddInitializer(const char* name, const OrtValue* val) noexcept;
+  Status AddInitializer(_In_z_ const char* name, _In_ const OrtValue* val) noexcept;
```

Comment:
>_In_z_ [](start = 24, length = 6)

What's the use for it internally?

Reply:
For Visual Studio Static Code Analyze.

Reply:
This change is to keep it consistent with the "AddConfigEntry" function in the same file.